src/analytics/ValoraAnalytics.ts

```diff
@@ -73,6 +80,7 @@ class ValoraAnalytics {
         throw Error('API Key not present, likely due to environment. Skipping enabling')
       }
       await Analytics.setup(SEGMENT_API_KEY, SEGMENT_OPTIONS)
+      await Statsig.initialize(STATSIG_API_KEY)
```

Comment:
Either use the provider, or use this.  Using both will get messy. If you want to handle the lifecycle yourself, then just do that here and use the apis on the global `Statsig`

If you were doing this and using `waitForInitialization`, this could cause an issue